standalone/standalone.go

```diff
@@ -118,6 +121,22 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	})
 }
 
+func registerExamplesHandler(mux *http.ServeMux, uiOpts *handlerOptions) {
+	examplesBlob, err := json.Marshal(uiOpts.examples)
+
+	mux.HandleFunc("/examples", func(w http.ResponseWriter, r *http.Request) {
+		if err == nil {
+			w.Header().Add("Content-Type", "application/json")
+			w.WriteHeader(200)
+			w.Write(examplesBlob)
+		} else {
+			w.Header().Add("Content-Type", "text/plain")
```

Comment:
I know I suggested this. So thank you for making the change. But I'm now wondering if we should actually push the marshalling to JSON all the way up to the call to `WithExamples` (when building the set of options) and just `panic` at that point if we can't marshal the examples. It's a little subtle and possibly bad user experience that we provide no feedback about bad examples provided until a browser tries to fetch them.

What do you think?

Reply:
good point, encoding errors should be a rare exception, if they happen then fail fast.